spec/components/breadcrumbs_spec.rb

```diff
@@ -17,27 +17,7 @@ def assert_link_with_text_in(selector, link, text)
   it "renders a single breadcrumb" do
     render_component(breadcrumbs: [{ title: 'Section', url: '/section' }])
 
-    assert_link_with_text_in('.gem-c-breadcrumbs__item:first-child', '/section', 'Section')
-  end
-
-  it "renders schema data" do
```

Comment:
We may need to keep this test. cc @vanitabarrett 
Is it broken in any way by the updated code?

Reply:
I think it's broken by removing lines 10-12 here: https://github.com/alphagov/govuk_publishing_components/pull/593/files/1cd3097e003479de96abf524900ff8c8ab7d0aa8#diff-0fdb3d47a2d7dd6a6eeb3693cbbf865cL10

Reply:
Thanks @vanitabarrett!
@DilwoarH I think we should keep the script; even though I agree is not good practice, but until we'll be able to chase it and find how can we improve it I would keep it to be backwards compatible.

Reply:
Hey @alex-ju - maybe I'm missing something but as far as I'm aware, this script was only used by the breadcrumbs, as we have deleted the code related to breadcrumbs - how would this make it backwards compatible?

Reply:
@alex-ju - I have added back the schema data